packages/calcite-ui-icons/LICENSE.md

```diff
@@ -0,0 +1,13 @@
+# License
```

Comment:
Sidebar: we should ensure all packages have a license and licensing info included.

Reply:
I remembered to do this when creating the monorepo! It looks like the tokens don't have the `license` field in package.json though, I'll add that in a separate PR.